Documentation of RSI API (GET PUSH PUT DELETE) Event (#25)
Akkadu/rsi-widgets: rsi-documentation/docs/interpreter-api/index.md

# Interpreters API

## Getting Started

In order to use our REST API you need to generate  an SDK key (From your [dashboard](https://rsi.akkadu.com/dashboard/events)) 

> When using our REST API you will need to use as `origin` the one you associated with your API key. However, the `localhost origin` is also whitelisted.

Reviewer: To successfully call our REST API endpoints you must be using the right `origin` that you have associated with your API key. However, for development the `localhost origin` is also whitelisted.

# Interpreters API

## Getting Started

> In order to use the Player SDK you require of a key (From your [rsi dashboard](https://rsi.akkadu.com)) and a room name from [Interpreter SDK](/interpretation-player/index.html) where you can create and handle your events.

Reviewer 2: ```suggestion
In order to use our REST API you need to generate of a SDK key (From your [dashboard](https://rsi.akkadu.com/dashboard/events)) 
```
Why would then necessary need a roomName to use that API? since they can create a roomName here as well, that's optional

> In addition we provide a RSI API where you can interact directly, integrate with your platform and automatize the process.

Reviewer 2: ```suggestion

```

> When we create a `sdk key` we require also to specify and origin. Only the origin and your localhost is whitelisted to make calls using this sdk key.  

Reviewer 2: ```suggestion
When using our REST API you will need to use as `origin` the one you associated with your API key. However the `localhost origin` is also whitelisted.

```

> In this section we will explain the different end points and how to work with them.

Reviewer 2: ```suggestion
### Endpoints: 
```

| Method  | End Point | Description |
| ------------- |:-------------:|:-------------:|

> | POST | `/event` | [Create a Event](/interpreter-api/docs.html#create-a-event) |

Reviewer 2: ```suggestion
| POST | `/event` | [Create an Event](/interpreter-api/docs.html#create-a-event) |
```

> | PUT | `/event` | [Update a Event](/interpreter-api/docs.html#update-an-event) |

Reviewer 2: ```suggestion
| PUT | `/event` | [Update an Event](/interpreter-api/docs.html#update-an-event) |
```

> | DELETE | `/event` | [Delete Event](/interpreter-api/docs.html#delete-a-event) |

Reviewer 2: ```suggestion
| DELETE | `/event` | [Delete an Event](/interpreter-api/docs.html#delete-a-event) |
```

> | GET | `/event` | [Get Event List](/interpreter-api/docs.html#get-event-list) |

Reviewer 2: ```suggestion
| GET | `/event` | [Get your Events](/interpreter-api/docs.html#get-event-list) |
```
